tests/conftest.py

```diff
@@ -98,6 +98,21 @@ def verify_cleanup():
     assert not threads
 
 
+@pytest.fixture(autouse=True)
```

Comment:
Should we autouse this?

Reply:
I don't see why not. I don't see any reason to run hashes extra times for the forced computational overhead that security requires, during tests.

It's just likely that we miss cases where a fixture/test pull in bcrypt work and we incure extra runtime for no real reason.

It's a bit hard to deduce when a test somewhere in it's tested code/library/fixture ends up using bcrypt stuff.

I don't mind moving the fixture to the tests/components/homeassistant/contest.py if we want to limit the scope. But I really think we should make it global.